iocage_lib/ioc_create.py

```diff
@@ -879,7 +879,9 @@ def create_install_packages(self, jail_uuid, location,
                     _callback=self.callback)
 
         # We will have mismatched ABI errors from earlier, this is to be safe.
-        pkg_env = {"ASSUME_ALWAYS_YES": "yes"}
+        pkg_env = {
+            **os.environ.copy(), "ASSUME_ALWAYS_YES": "yes"
```

Comment:
This will leak the users environment during execution (ssh and such), we should keep this minimal and add just the two proxy variables.